src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -59,13 +59,13 @@
       "Size": 9591
     },
     "assemblies/UnnamedProject.dll": {
-      "Size": 3560
+      "Size": 3555
     },
     "classes.dex": {
       "Size": 348440
     },
     "lib/arm64-v8a/libmonodroid.so": {
-      "Size": 484512
+      "Size": 510920
```

Comment:
Seems odd that `libmonodroid.so` is ~26KB larger, yet lots of the new C++ code is not even compiled.  *Weird*.

Reply:
A lot more code is inlined now. It's a bit surprising it generates that much of a difference, tbh.